upload-cloud-storage/src/upload-helper.ts

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Storage, UploadResponse } from '@google-cloud/storage';
+import * as path from 'path';
+import { getFiles } from './util';
+
+export class UploadHelper {
+  readonly storage: Storage;
+  constructor(storage: Storage) {
+    this.storage = storage;
+  }
+
+  /**
+   * Uploads a file to a bucket
```

Comment:
Missing punctuation here and throughout.

Reply:
Fixed